address_test.go

```diff
@@ -0,0 +1,35 @@
+package maddy
+
+import (
+	"net"
+	"reflect"
+	"testing"
+)
+
+var cases = []Address{
```

Comment:
Nitpicking, but can you move that slice into TestStandardizeAddress so we will not have too generically named variable ("cases", which cases?) in the global scope?

Reply:
Done!